ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/beta-banner/beta-banner.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { async, ComponentFixture, TestBed, fakeAsync, tick } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { Component } from '@angular/core';
+import { BetaBannerComponent } from './beta-banner.component';
+import 'rxjs/add/operator/filter';
+import { RouterTestingModule } from '@angular/router/testing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+
+@Component({ selector: 'app-mock-component', template: '' })
+class Mock1Component {
+}
+
+@Component({ selector: 'app-mock-component2', template: '' })
+class Mock2Component {
+}
+
+const routes =
+  [
+    { path: 'sub-component1', component: Mock1Component },
+    { path: 'sub-component2', component: Mock2Component }
+  ];
+
+describe('FooterComponent',
+  () => {
+    let component: BetaBannerComponent;
+    let fixture: ComponentFixture<BetaBannerComponent>;
+    let router: Router;
+
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        declarations: [
+          BetaBannerComponent,
+          Mock1Component,
+          Mock2Component
+        ],
+        imports: [
+          RouterTestingModule.withRoutes(routes)
+        ],
+        schemas: [NO_ERRORS_SCHEMA]
+
+      })
+        .compileComponents();
+    }));
+
+    beforeEach(() => {
+      router = TestBed.get(Router);
+      fixture = TestBed.createComponent(BetaBannerComponent);
+      component = fixture.componentInstance;
+      fixture.detectChanges();
+    });
+
+    it('should create',
```

Comment:
I think you can remove this, it'll be tested implicitly by the other test otherwise and saves you some runtime

Reply:
Removed it